Oracle/med_admin.sql

```diff
@@ -155,3 +155,4 @@ set end_time = sysdate, records = (select count(*) from med_admin)
 where task = 'med_admin'
 /
 select records from cdm_status where task = 'med_admin'
+/
```

Comment:
https://github.com/kumc-bmi/i2p-transform/blob/a1fd9deee02adf01bb9b4b5571319614873590ec/Oracle/diagnosis.sql 's last line does not end with `/` what does it do ?

Reply:
diagnosis.sql is OK without the `/` at the end.

I was fixing problems where we had both `;` and `/` so that `/` was treated as a separator and `;` got included in the statement, which Oracle didn't like.

Adding this `/` probably wasn't needed.